internal/osutils/autostart/autostart.go

```diff
@@ -1,50 +1,78 @@
 package autostart
 
 import (
+	"github.com/ActiveState/cli/internal/constants"
 	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
 )
 
 const ConfigKeyDisabled = "SystrayAutoStartDisabled"
```

Comment:
```suggestion
const ConfigKeyDisabled = "systray.autostarted.disabled"
```
We should format these consistently.

Reply:
Also this config key seems too generic now. It says systray but effectively would also be used for the svc?